README.md

```diff
@@ -13,11 +13,15 @@
 </div>
 <div align="center">
 
-[![Build Status][WorkflowBadge]][WorkflowUrl]
-[![Sponsor][SponsorBadge]][SponsorUrl]
-[![Patreon][PatreonBadge]][PatreonUrl]
-[![Discord][DiscordBadge]][DiscordUrl]
-[![Twitter][TwitterUrl]][TwitterBadge]
+
+[![Sponsor](https://img.shields.io/badge/Sponsor-100000?style=for-the-badge&logo=github&logoColor=white)](https://github.com/sponsors/medvednikov)
+[![Patreon](https://img.shields.io/badge/Patreon-F96854?style=for-the-badge&logo=patreon&logoColor=white)](https://patreon.com/vlang)
+[![Twitter](https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/v_language)
+[![Discord](https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white)](https://discord.gg/vlang)
+[![GitHub branch checks state](https://img.shields.io/github/checks-status/vlang/v/master?style=for-the-badge)](https://github.com/vlang/v/master)
+<!-- failed to check workflow name -->
+<!-- 
+[![Build Statu](https://img.shields.io/appveyor/build/vlang/v?style=for-the-badge)](https://github.com/vlang/v/commits/master) -->
```

Comment:
I didn't find the workflow name to add it here

`
user
vlang
e.g. actions
/
repo
v
toolkit
/
workflow
toolkit-unit-tests
/
branch
master

`


https://img.shields.io/github/workflow/status/vlang/v/WORKFLOWNAME/master